server-tests.php

```diff
@@ -736,10 +720,6 @@ function test_sort($a, $b) {
 	}
 
 	function checkRequirements() {
-		if (version_compare(phpversion(), "5.0") < 0) {
-			$this->writemsg(REQ_PHP_VERSION);
-			exit;
-		}
```

Comment:
I think the version check should remain in place, just with an updated version number (presumably `5.4` now?)

Note that `run-test.php` is not necessarily run using the same binary it tests, as such it makes sense to support more versions than just the absolutely most recent pre-release. (5.4 onwards is quite enough though.)